src/plugins/charts/public/services/mapped_colors/mapped_colors.ts

```diff
@@ -26,16 +23,13 @@ export class MappedColors {
   private _oldMap: any;
   private _mapping: any;
 
-  constructor(
-    private uiSettings?: CoreSetup['uiSettings'],
-    private colorPaletteFn: (num: number) => string[] = createColorPalette
-  ) {
+  constructor(private colorPaletteFn: (num: number) => string[] = createColorPalette) {
     this._oldMap = {};
     this._mapping = {};
   }
 
   private getConfigColorMapping(): Record<string, string> {
-    return _.mapValues(this.uiSettings?.get(COLOR_MAPPING_SETTING) || {}, standardizeColor);
+    return _.mapValues({}, standardizeColor);
```

Comment:
This will return an empty object now.
If that's the expected result, why just return it?

```suggestion
    return {};
```

And if so the `standardizeColor` can be removed altogether.

Reply:
If that is the case, I also believe the entire class can be removed, because it relies on this method to get the color for a specific key.

Reply:
and probably we can also remove the `syncColors` functionality that relies on this method

Reply:
I simplified the MappedColors class but we cannot remove it completely because still need `syncColors` functionality, unless we decide to remove it from dashboard.

<img width="1183" alt="Screenshot 2024-11-11 at 16 28 01" src="https://github.com/user-attachments/assets/2c6e9598-9324-462e-bc9d-bf40c4bf36ca">